jbpm-wb-forms/jbpm-wb-forms-client/src/main/java/org/jbpm/workbench/forms/client/display/process/AbstractStartProcessFormDisplayer.java

```diff
@@ -203,7 +203,7 @@ public boolean error(Message message,
                                  Throwable throwable) {
                 String notification = Constants.INSTANCE.UnexpectedError(throwable.getMessage());
                 errorPopup.showMessage(notification);
-                return true;
+                return false;
```

Comment:
@bxf12315 have you considered removing _getUnexpectedErrorCallback_ ? I think that could be a good approach to fallback to the default error handler instead ( _DefaultWorkbenchErrorCallback_ ), wdyt?